kernelci/build.py

```diff
@@ -832,12 +833,13 @@ def push_kernel(kdir, api, token, install='_install_'):
     *kdir* is the path to the kernel source directory
     *api* is the URL of the KernelCI backend API
     *token* is the token to use with the KernelCI backend API
-    *install* is the path to the installation directory inside kdir
+    *install* is the path to the installation directory
 
     The returned value is True if it was done successfully or False if an error
     occurred.
     """
-    install_path = os.path.join(kdir, install)
+    if not install_path:
+        install_path = os.path.join(kdir, '_install_')
```

Comment:
So the default value is actually still inside `kdir`.  I think it's fine, but maybe the docstring should be clearer about the default behaviour.  Maybe something like this:
```
*install_path* is the path to the installation directory, or _install_ inside kdir by default
```

Reply:
Yes, the default behavior should not have changed with these patches. Indeed, I did not do a great job at documenting this. I'll improve this

Reply:
OK I see you've explained the default path in the command line argument help, which is more useful for the user so that's fine.

The default path is however actually implemented in this function, and that's not explained in the docstring.  Maybe the default path should be all handled by `kci_build` and there should be no default value in this function.  Something to improve in another PR I guess, it's not a big deal.